modules/cf-core/src/store.ts

```diff
@@ -30,6 +27,49 @@ export class Store {
     private readonly storeKeyPrefix: string
   ) {}
 
+  // TODO: remove if store is added to Context type
+  public static async getMultisigAddressWithCounterpartyFromMap(
```

Comment:
make this a non-static util function

also, what would be the motivation for adding the store to the `Context` type?

Reply:
Idk the motivation, I think cf did that in the store refactor that we avoided pulling in (I just moved this fn to the top so it looks like I wrote this)